### DISCUSSION TITLE
Integrating new App

### DISCUSSION THREAD

**Discussion by VHolstein** [carissalow/rapids#189] [Ideas]
Hi!

I am currently working with GPS data from an app that is not supported by RAPIDS yet. To process the data using the Barnett pipeline I want to integrate a container and format.yaml that support the app. I wanted to ask how the folder and csv structure for the aware_csv container looks like, as I would adapt the scripts based on these.

Thanks :)

**Comment by JulioV** [comment]
Hi! That's great to hear. So all of our data streams containers and format files are in this [folder](https://github.com/carissalow/rapids/tree/master/src/data/streams), including `aware_csv`. This doc [page](https://www.rapids.science/1.8/datastreams/add-new-data-streams/) should contain most of what you need. If something is missing or you have any questions, let us know!

**Reply by VHolstein** [reply]
I made a fork and wrote a basic container and format script - to debug the script I'll need to install my repo fork. For this, I'll need to modify the docker script (as I'm planning to install it as a docker), so I wanted to ask where and how I could modify the script to use my fork?

**Reply by JulioV** [reply]
The Docker file is [here](https://github.com/carissalow/rapids/blob/master/Dockerfile) but you could actually test your fork by adding it as another remote in the repo clone of the Docker deployment that you already have, and the pulling/checking our your new branch. So no need to recreate the container.

**Reply by VHolstein** [reply]
Great, thank you! I will use that workaround - one more question: for reference, I'd like to process the GPS data using the Barnett pipeline with the aware_csv data stream as well. Is there an example csv file for aware_csv GPS data available?

**Reply by JulioV** [reply]
Yeah this is an example Database

https://osf.io/skqfv/files/osfstorage
